### PR TITLE
update language to use allowlist instead of whitelist

### PR DESCRIPTION
**Pull request by chris-tbd**
## What type of PR is this? (check all applicable)

- [ ] ♻️ Refactor
- [ ] ✨ New Feature
- [ ] 🐛 Bug Fix
- [ x] 📝 Documentation Update
- [ ] 👷 Example Application
- [ ] 🧑‍💻 Code Snippet
- [ ] 🎨 Design
- [ ] 📖 Content
- [ ] 🧪 Tests
- [ ] 🔖 Release
- [ ] 🚩 Other

## Description

Updated usage of "whitelist" to "allowlist" in tbDEX guides

## Related Tickets & Documents

Resolves #

## Added code snippets?
- [ ] 👍 yes
- [ x] 🙅 no, because they aren't needed

## Added tests?

- [ ] 👍 yes
- [ x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### No tests? Add a note
Just updating language

## Added to documentation?

- [ ] 📜 readme
- [ ] 📜 contributing.md
- [ ] 📓 general documentation
- [ x] 🙅 no documentation needed

### No docs? Add a note
Updating language, not changing any actual content

### CODE REVIEW

File: site/docs/tbdex/wallet/allowlist-pfis.mdx [hunk 1]
```diff
@@ -2,15 +2,15 @@
...
 ---
 
-# Whitelisting PFIs
+# Allowlisting PFIs
 
-While tbDEX is a permissionless network where any PFI can participate, you may want to curate your own list of PFIs in which you engage with. We refer to this as **whitelisting**.
+While tbDEX is a permissionless network where any PFI can participate, you may want to curate your own list of PFIs in which you engage with. We refer to this as **allowlisting**.
 
-Whitelisting is the process of approving or recognizing specific PFIs as trusted entities. 
+allowlisting is the process of approving or recognizing specific PFIs as trusted entities.
```
Comment (angiejones) [inline site/docs/tbdex/wallet/allowlist-pfis.mdx:9]:
```suggestion
Allowlisting is the process of approving or recognizing specific PFIs as trusted entities. 
```

File: site/docs/tbdex/wallet/allowlist-pfis.mdx [hunk 2]
```diff
@@ -27,9 +27,9 @@ const isPFI = pfiDid.document.service.some(service => service.type === 'PFI');
 Determine if there are certain credentials you require from a PFI, such as regulatory compliance status. 
 These may or may not be provided in the form of verifiable credentials.
 
-## Whitelist PFI
+## allowlist PFI
```
Comment (angiejones) [inline site/docs/tbdex/wallet/allowlist-pfis.mdx:30]:
```suggestion
## Allowlist PFI
```

File: site/docs/tbdex/key-concepts.mdx
Comment (angiejones) [inline site/docs/tbdex/key-concepts.mdx:1]: this one is part of another PR. I put some comments on that one

**Comment by angiejones** [comment]
Really appreciate the thoughtfulness of the word change. Wondering if this will come off as contradictory to tbDEX being a permissionless network though.

**Comment by chris-tbd** [comment]
Should we use a phrase like "Add Searchable PFIs," "Add PFIs," or "Enable PFIs" instead? Because I do see how allowlist/whitelist is confusing since we aren't actually "allowing" the PFI to interact with the wallet, we're just enabling it

File: site/docs/tbdex/key-concepts.mdx
Comment (angiejones) [reply site/docs/tbdex/key-concepts.mdx:1]: @chris-tbd can you remove this file from the PR? I think you should be able to do `checkout site/docs/tbdex/key-concepts.mdx`

Comment (angiejones) [inline site/docs/tbdex/key-concepts.mdx:1]: remove this file from PR

**Review by angiejones — CHANGES_REQUESTED** [review]

**Review by angiejones — APPROVED** [review]

**Pull request merged**